packages/react/src/components/link/Link.tsx

```diff
@@ -18,7 +18,7 @@ export type LinkProps = Omit<
   /**
    * Link content
    */
-  children: string;
+  children: React.ReactNode;
```

Comment:
This would probably be better off as `children: React.ReactChildren`.

Reply:
That sounded good but after testing it, it doesn't accept string typed children :S `Type 'string' is not assignable to type 'ReactNode & ReactChildren'`